styles/hydrogen.less

```diff
@@ -79,6 +79,7 @@
 
         .bubble-output-container {
             padding: 3px;
+            overflow: scroll;
```

Comment:
If this is only going to have an effect on watches, we could put this in the `.watch-sidebar` section: https://github.com/willwhitney/hydrogen/blob/result-watch/styles/hydrogen.less#L185